packages/commonwealth/server/migrations/20250516150000-remove-group-ids-from-topic.js

```diff
@@ -3,11 +3,15 @@
 /** @type {import('sequelize-cli').Migration} */
 module.exports = {
   async up(queryInterface) {
-    await queryInterface.sequelize.transaction(async (transaction) => {
-      await queryInterface.removeColumn('Topics', 'group_ids', {
-        transaction,
+    try {
+      await queryInterface.sequelize.transaction(async (transaction) => {
```

Comment:
???

Reply:
this is to avoid the migration crashing the CI after the col has been removed... not sure why but it happened to me locally and to Ilija on beta

Reply:
Probably because you re-ordered the migration after having already run it with the old name. Not sure why that would happen on beta but ye